src/task4feedback/simulator/schedulers/parla/architecture.py

```diff
@@ -633,9 +780,17 @@ def complete_task(
         self, scheduler_state: SystemState, event: TaskCompleted
     ) -> List[EventPair]:
         objects = scheduler_state.objects
+
         task = objects.get_task(event.task)
         current_time = scheduler_state.time
 
+        next_events: List[EventPair] = []
+        self.success_count += 1
```

Comment:
..I don't think it matters because a "task completion failure" means the program crashes anyway. My preference is to only increment "success" after it's been completed though